api/v1beta1/conditions.go

```diff
@@ -43,4 +43,13 @@ const (
 	InvalidBackendErrorMessageSplit = "The GlanceAPI layout type: split cannot be used in combination with File and NFS backend"
 	// InvalidBackendErrorMessageSingle
 	InvalidBackendErrorMessageSingle = "The GlanceAPI layout type: single can only be used in combination with File and NFS backend"
+
+	// TopologyConfigReadyInitMessage
+	TopologyConfigReadyInitMessage = "Topology config create not started"
+	// TopologyConfigReadyMessage
+	TopologyConfigReadyMessage = "Topology config create completed"
+	// TopologyConfigReadyErrorMessage
+	TopologyConfigReadyErrorMessage = "Topology config create error occurred %s"
+	// TopologyConfigReadyCondition Status=True condition which indicates a CR exists and is referenced by the Glance
+	TopologyConfigReadyCondition condition.Type = "TopologyConfigReady"
```

Comment:
maybe these conditions should go in the infra-operator PR as part of the topology api to be used by any consumer

Reply:
Right, in a world where this kind of change should be spread across N operators I think best to have these conditions in infra-operator, I agree.
I'll work out this part (probably tomorrow), to make sure we can minimize the code we need in each operator to handle topologies.